tests/test_TUDPTransport.py

```diff
@@ -13,10 +13,8 @@
 # limitations under the License.
 
 from __future__ import absolute_import
-
-import mock
+# import mock  # noqa: F401
 import unittest
-
```

Comment:
Let's keep this blank line to separate the standard imports from out own modules